### DISCUSSION TITLE
coloring system file types

### DISCUSSION THREAD

**Discussion by flesti** [doublecmd/doublecmd#774] [General]
There's only 2 or 3 category attributes like "Dir" "Hidden" "Compressed". Where to find the rest, like "symbolic links" etc.

**Comment by alexx2000** [comment]
You can find symbolic link using 'l*' attribute mask, [see](https://doublecmd.github.io/doc/en/configuration.html#ConfigColorFiles). Also you can use `search templates`, press 'filter' button to open it configuration.
